UPGRADE-6.3.md

```diff
@@ -379,6 +379,7 @@ You should remove all width/height settings in CreateModalXY calls if it does no
 - loadStandaloneDocumentManager
 - PublishViaAjaxCallback()
 - src/CoreBundle/Resources/public/javascript/mainNav.js
+- showMLTField() - use CHAMELEON.CORE.MTTableEditor.switchMultiSelectListState(iFrameId, url) instead.
```

Comment:
Our best friend, the alphabetic order .. :-) (one line up, please)

Reply:
done